src/test/url-handling.test.js

```diff
@@ -46,3 +46,49 @@ describe('selectedThread', function () {
     expect(urlStateReducers.getSelectedThreadIndex(store.getState())).toBe(2);
   });
 });
+
+describe('threadOrder and hiddenThreads', function () {
+  const profile: Profile = processProfile(exampleProfile);
```

Comment:
nit: put this line in `storeWithSearch` so that we get a new profile object for each test.

(I know `exampleProfile` isn't new for each test, which is problematic because `processProfile` actually mutates the argument... but that's for later, baby steps)